### ISSUE TITLE
integrate with ownCloud's new tagging mechanism

### ISSUE THREAD

**Issue by georgehrke** [owncloud/calendar#478]

**Comment by aspdye** [comment]
Do you simply want to adapt this for the calendar, or do you want to integrate it, so that searching for tags in Files will also show events? 

Are there any possibility to push the tags via CalDAV to the clients? I don't think so 😁

**Comment by georgehrke** [comment]
> Are there any possibility to push the tags via CalDAV to the clients?

My plan is to "sync" the tags with the categories field in an event.

**Comment by raimund-schluessler** [comment]
"Tags" are called "Categories" in CalDAV: https://tools.ietf.org/html/rfc5545#section-3.8.1.2

So setting a tag in owncloud should probably set the category for the event and vice versa. This would require work in core, but probably made it easier for the Calendar as it would only have to perform a CalDAV request and core handles the category/tag setting. This would also allow other CalDAV clients to integrate into the oc Tag feature.

@DeepDiver1975 Would this work?

Edit: @georgehrke Was faster ;)

**Comment by aspdye** [comment]
Good call. I think this would work with some work in core (dav) but this can be first integrated with 9.1 (no back port 😁) - So I think 1.3-next as a milestone is a bit to optimistic 😅

**Comment by Mannshoch** [comment]
Maybe a future public sharing from the Calendar could also base on tagging like with a "Public" Tag:
https://github.com/owncloud/core/issues/22735
